### PR TITLE
#14: Implement ToolCommandlet for Azure CLI

### PR DESCRIPTION
**Pull request by MattesMrzik**
#14: Implemented ToolCommandlet for Azure CLI.

### CODE REVIEW

File: cli/src/main/java/com/devonfw/tools/ide/tool/az/Azure.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  }
+
+  @Override
+  public void postInstall() {
+
+    super.postInstall();
+
+    EnvironmentVariables variables = this.context.getVariables();
+    EnvironmentVariables settingsVariables = variables.getByType(EnvironmentVariablesType.WORKSPACE);
+    // TODO oder EnvironmentVariablesType.CONF
```
Comment (hohwille) [inline cli/src/main/java/com/devonfw/tools/ide/tool/az/Azure.java:34]: yep, needs to go to `conf/ide.settings`.

File: cli/src/main/java/com/devonfw/tools/ide/tool/az/Azure.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    // TODO I think there is nothing written to a file, just the env var exported and thereby only active in current
+    // session. Should this env var be written to a file?
```
Comment (hohwille) [inline cli/src/main/java/com/devonfw/tools/ide/tool/az/Azure.java:36]: Simply do the change and then call `save` method on the according variables.
FYI: Only `EnvironmentVariablesPropertiesFile` implements `save` properly. So currently doing `variables.save()` would have no effect. Maybe we should throw an `UnsupportedOperationException` in this case so the developer does not get mistaken...

Comment (hohwille) [reply cli/src/main/java/com/devonfw/tools/ide/tool/az/Azure.java:36]: See also #78

**Comment by MattesMrzik** [comment]
Why hasn't this:
`[hohwille](https://github.com/hohwille) linked an issue [yesterday](https://github.com/devonfw/IDEasy/pull/99#event-10681228055) that may be closed by this pull request`
happened automatically? It seems that I can't manually link these as I have no gear-symbol at the Development section on the right.

**Comment by hohwille** [comment]
> Why hasn't this: ... happened automatically? 

Read out Teams channel, where I linked the solution.

**Review by hohwille — APPROVED** [review]
@MattesMrzik Thanks for your PR. Solved nicely and perfect 👍

**Pull request merged**